src/Microsoft.TestPlatform.CommunicationUtilities/Resources/xlf/Resources.cs.xlf

```diff
@@ -13,6 +13,11 @@
         <target state="translated">Vzdálený hostitel vynutil ukončení existujícího připojení.</target>
       </trans-unit>
       <trans-unit id="AbortedTestDiscovery">
+        <source>The active test discovery was aborted.</source>
+        <note></note>
+        <target state="translated">Aktivní zjišťování testu se přerušilo.</target>

```

Comment:
Added the `.xlf` as already translated as I am only adding a resource removing a sub sentence.